app/controllers/admin/appointment_summaries_controller.rb

```diff
@@ -0,0 +1,21 @@
+module Admin
+  class AppointmentSummariesController < ::ApplicationController
+    before_action :authenticate_user!
+
+    def index
+      redirect_to :root unless current_user.admin?
```

Comment:
might be nicer to wrap the two bit of authentication together into an `authenticate_admin!` method